Filter/RequestFilter.php

```diff
@@ -39,6 +64,34 @@ public function filter(Request $request)
                 $filteredValues->set($filter, $value);
             }
         }
-        return $filteredValues;
+        $this->criteria = $filteredValues->all();
+
+        // filter the request parameters to find some orders parameters
+        if ($request->get('sort') && $request->get('order')) {
+            // if sort and order parameters are present in the request, we use them
+            $this->orders = [
+                $request->get('sort') => $request->get('order')
+            ];
+        }
+    }
+
+    public function getCriteria()
+    {
+        return $this->criteria;
+    }
+
+    public function getOrder()
+    {
+        return $this->orders;
+    }
+
+    public function getMaxPerPage()
+    {
+        return $this->maxPerPage;
+    }
+
+    public function getCurrentPage()
+    {
+        return 1;
```

Comment:
Yess, this is a default value that does not be set in the default RequetFilter, only in the PagerFantaRequestFilter